.github/workflows/posix-deps-apt.sh

```diff
@@ -19,6 +19,7 @@ apt-get -yq install \
     libreadline6-dev \
     libsqlite3-dev \
     libssl-dev \
+    libnsl-dev \
```

Comment:
Retain alphabetical order:
```suggestion
    libnsl-dev \
    libreadline6-dev \
    libsqlite3-dev \
    libssl-dev \
```